core/string/translation.cpp

```diff
@@ -702,6 +702,7 @@ void TranslationServer::setup() {
 	pseudolocalization_skip_placeholders_enabled = GLOBAL_DEF("internationalization/pseudolocalization/skip_placeholders", true);
 
 #ifdef TOOLS_ENABLED
+	ProjectSettings::get_singleton()->set_custom_property_info(PropertyInfo(Variant::STRING, "internationalization/locale/test", PROPERTY_HINT_LOCALE_ID, ""));
```

Comment:
This should be in a separate PR as it is a functional change unlike everything else

Reply:
OK. It might make sense to batch them just like typos, so we have one "fix X typos" PR and another "fix X UI papercuts" like propertyinfo and basic/advanced project settings flags, which can be reviewed separately.